src/tests/load-function-module.test.ts

```diff
@@ -10,25 +9,33 @@ Deno.test("LoadFunctionModule function", async (t) => {
   const functionsDir = `${fixturesDir}/functions`;
 
   await t.step(
-    "should throw if a callback_id in the payload does not exist",
+    "should return null if no files are provided",
     async () => {
-      await assertRejects(
-        async () => {
-          const payload = generatePayload("funky");
-          payload.body.event.function.callback_id = "";
-          return await LoadFunctionModule(functionsDir, payload);
-        },
-        Error,
-        "No callback_id provided in payload!",
-      );
+      const fnModule = await LoadFunctionModule([]);
+      assertEquals(fnModule, null);
     },
   );
 
+  await t.step("should load the correct file when given multiple", async () => {
```

Comment:
💭 Wondering if we should have two tests here:

1. Validates that a `ts` file will be used instead of a `js` file when they both exist.
2. Validates that it'll use the existing `js` file over a `ts` file that doesn't exist.

Reply:
Good call, I'll add this as an issue and flag it good first issue (and also turn on issues for this repo)

Reply:
https://github.com/slackapi/deno-slack-runtime/issues/16